### DISCUSSION TITLE
Vulnerablity Report #1 Security Misconfiguratioon

### DISCUSSION THREAD

**Discussion by rhonnysharma** [clearpool-finance/community#17] [Ideas]
Bug Description:
The Bimi record is missing from the email header, resulting in the failure to display a brand logo in the recipient's email client. This bug affects the visual representation of emails and compromises brand authenticity and identification.

Steps to Reproduce:

1) Navigate this url https://dmarcly.com/tools/bimi-record-checker 
and enter your domain name :clearpool.finance

Scenario:
Hi, let me clear your doubts . 

What does a BIMI record do?A BIMI Record is a type of DNS Record used to display a company's logo inside an email inbox if the email is trustworthy

So why is the brand logo so important?

Every company has a different brand logo which makes it unique because the brand logo signifies your product , website , company infrastructure etc.

when a email come from your company the brand logo  makes trust and loyalty to the user  and specify the real company 

To fix this bug follow this - https://mxtoolbox.com/dmarc/bimi/how-to-create-bimi-record

Expected Result:
The recipient's email client should display the brand's logo next to the sender's name or email address, providing visual authentication and identification.

Actual Result:
The recipient's email client does not display the brand's logo. Instead, a generic placeholder or no image is shown next to the sender's name or email address.

Impact:
The absence of the brand's logo due to the missing Bimi record affects the visual representation of emails, potentially leading to reduced brand recognition and user trust. It also undermines the purpose of Bimi, which is to provide visual authentication and identification for email recipients.

**Comment by zolotokrylin** [comment]
This is not a bug, and it is also a duplicate of this thread: https://github.com/orgs/clearpool-finance/discussions/15
